code/modules/spells/targeted/pacify.dm

```diff
@@ -0,0 +1,80 @@
+/spell/targeted/pacify
+	name = "Pacify"
+	desc = "Generates a burst of calming energies which inhibit hostile behavior in living beings. The caster is slightly affected by channeling these energies."
+	user_type = USER_TYPE_WIZARD
+	specialization = DEFENSIVE
+
+	charge_max = 45 SECONDS
+	cooldown_reduc = 15 SECONDS
+	cooldown_min = 15 SECONDS
+
+	spell_levels = list(Sp_SPEED = 0, Sp_RANGE = 0, Sp_POWER = 0)
+	level_max = list(Sp_TOTAL = 3, Sp_SPEED = 2, Sp_RANGE = 1, Sp_POWER = 2)
+
+	spell_flags = WAIT_FOR_CLICK
+	invocation = "YUKKRI SHEETI NAY"
+	invocation_type = SpI_SHOUT
+	range = 6
+
+	max_targets = 1
+
+	hud_state = "wiz_pacify"
+	var/power = 1
```

Comment:
Why not using `spell_levels[Sp_POWER]` directly instead

Reply:
Fixed